src/ape_run/_cli.py

```diff
@@ -16,6 +18,19 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs, result_callback=self.result_callback)
         self._namespace = {}
 
+    def invoke(self, ctx: Context) -> Any:
+        try:
+            return super().invoke(ctx)
+        except Exception:
+            if ctx.params["interactive"]:
+                # Print the exception trace and then launch the console
+                err_info = traceback.format_exc()
+                click.echo(err_info)
+                self._launch_console()
+            else:
+                # Don't handle error - raise exception as normal.
+                raise
```

Comment:
NOTE: When you raise without arguments, the interpreter looks for the last exception raised and handled. It then acts the same as if you used raise with the most recent exception type, value and traceback.

https://stackoverflow.com/questions/18001721/raise-with-no-argument